sql/core/src/test/resources/sql-tests/inputs/interval.sql

```diff
@@ -222,7 +222,17 @@ select a * 1.1 from values (interval '-2147483648 months', interval '2147483647
 select a / 0.5 from values (interval '-2147483648 months', interval '2147483647 months') t(a, b);
 
 -- interval support for csv and json functions
-SELECT from_csv('1, 1 day', 'a INT, b interval');
-SELECT to_csv(named_struct('a', interval 32 month, 'b', interval 70 minute));
-SELECT from_json('{"a":"1 days"}', 'a interval');
-SELECT to_json(map('a', interval 25 month 100 day 130 minute));
+SELECT
+  from_csv('1, 1 day', 'a INT, b interval') a,
```

Comment:
I think we only need 4 cases:
1. from_csv with a constant
2. to_csv with a constant
3. from_csv(to_csv(...))
4. to_csv(from_csv(...))

same to json